articles/operator-nexus/troubleshoot-cluster-heartbeat-connection-status-disconnected.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Troubleshoot Azure Operator Nexus Cluster Heartbeat Connection Status shows Disconnected
+description: Provide steps to investigate and possibly resolve circumstances that are preventing the Cluster from sending heartbeats to the Cluster Manager.
+ms.service: azure-operator-nexus
+ms.custom: troubleshooting
+ms.topic: troubleshooting
+ms.date: 10/09/2024
+ms.author: omarrivera
+author: omarrivera
+---
+# Troubleshoot Azure Operator Nexus Cluster Heartbeat Connection Status shows Disconnected
+
+This guide attempts to provide steps to troubleshoot a Cluster is shown to have `clusterConnectionStatus` with a value of `Disconnected`.
+
+> [!CAUTION]
+> The `ClusterConnectionStatus` is likely a symptom or signal and not the root cause and this guide will not be able to provide answers for all scenarios.
+> The focus and purpose of this guide is to provide common issues and signals that can be inspected to determine where the issue might be.
+
+## Understanding the Issue
+
+Cluster Managers ensure continuous Cluster network connectivity through a heartbeat agent running within the target Cluster.
+The cluster-heartbeat agent sends periodic HTTP messages to the Cluster Manager and expects an acknowledgment response as well.
+A Cluster has the property `ClusterConnectionStatus` which is set to the value `Connected` as the heartbeats are continuously received and acknowledged.
+
+The `ClusterConnectionStatus` becomes `Connected` once the cluster is in a healthy state and network connectivity issues are resolved.
+If the Cluster is expected to be healthy but the `ClusterConnectionStatus` remains in `Disconnected` state [contact support] after following the steps in this guide.
+
+> [!IMPORTANT]
+> `ClusterConnectionStatus` is **not** the same as Arc Connected Kubernetes Clusters.
+
```

Comment:
Can we include metric also to monitor cluster connection status. 

like below: Avg Cluster Connection Status for <<cmname>> by Cluster Name where Cluster Name = <<'clustername'>>